multiqc/modules/bclconvert/bclconvert.py

```diff
@@ -182,7 +185,7 @@ def __init__(self):
         )
 
         # Add section with undetermined barcodes
-        if self.num_demux_files == 1:
+        if create_undetermined_barplots:
```

Comment:
I think we still want to do it by default if only a single demux file.. I can't see a reason _not_ to do it in this case.

Reply:
Addressed in  e9c0e26